src/app/components/layouts/column-container/terra-two-columns-container.component.ts

```diff
@@ -37,11 +53,61 @@ export class TerraTwoColumnsContainerComponent
         this.rightColumn = this.leftRightColXS() + this.rightColMD() + this.rightColLG();
     }
 
-    constructor()
+    private readonly basePath:string;
+
+    constructor(private route:ActivatedRoute,
+                private router:Router)
     {
+        this.basePath = router.routerState.snapshot.url;
+        this.setColumnHidden('left');
+
+        let subscribable:Observable<Event> = this.router.events.filter((event:RouterEvent) =>
+        {
+            return event instanceof NavigationEnd && event.urlAfterRedirects.startsWith(this.basePath);
+        });
+
+        this.subscription = subscribable.subscribe((event:NavigationEnd) =>
+        {
+            this.leftColumn = this.leftRightColXS() + this.leftColMD() + this.leftColLG();
+            this.rightColumn = this.leftRightColXS() + this.rightColMD() + this.rightColLG();
```

Comment:
Das muss hier eigentlich nicht ausgeführt werden, das wird ja anschließend in `this.setColumnHidden()` aktualisiert, oder?